src/App.js

```diff
@@ -1,4 +1,5 @@
-import Logo from './logo.png'
+import Logo from './logo.png';
+import fb from './fb_icon.svg';
```

Comment:
Can we keep this consistent pls? Also a verbose import would help in the future to know what's being imported.
```suggestion
import Fb from './fb_icon.svg';
// or
import FacebookLogo from './fb_icon.svg';
```